internal/eval/eval_test.go

```diff
@@ -44,79 +44,79 @@ func setupTest(store *dstore.Store) *dstore.Store {
 func TestEval(t *testing.T) {
 	store := dstore.NewStore(nil, nil)
 
-	testEvalMSET(t, store)
-	testEvalECHO(t, store)
-	testEvalHELLO(t, store)
-	testEvalSET(t, store)
-	testEvalGET(t, store)
-	testEvalGETEX(t, store)
-	testEvalDebug(t, store)
-	testEvalJSONARRTRIM(t, store)
-	testEvalJSONARRINSERT(t, store)
-	testEvalJSONARRPOP(t, store)
-	testEvalJSONARRLEN(t, store)
-	testEvalJSONDEL(t, store)
-	testEvalJSONFORGET(t, store)
-	testEvalJSONCLEAR(t, store)
-	testEvalJSONTYPE(t, store)
-	testEvalJSONGET(t, store)
-	testEvalJSONSET(t, store)
-	testEvalJSONNUMMULTBY(t, store)
-	testEvalJSONTOGGLE(t, store)
-	testEvalJSONARRAPPEND(t, store)
-	testEvalJSONRESP(t, store)
-	testEvalTTL(t, store)
-	testEvalDel(t, store)
-	testEvalPersist(t, store)
-	testEvalEXPIRE(t, store)
-	testEvalEXPIRETIME(t, store)
-	testEvalEXPIREAT(t, store)
-	testEvalDbsize(t, store)
-	testEvalGETSET(t, store)
-	testEvalHSET(t, store)
-	testEvalHMSET(t, store)
-	testEvalHKEYS(t, store)
+	// testEvalMSET(t, store)
```

Comment:
Should we have these test functions commented out ?

Reply:
Oh shoot, I thought to uncomment them before making a commit. I made a mistake here. Let me quickly correct it.